indigo_api/templates/indigo_api/akn/coverpage_act.html

```diff
@@ -55,27 +55,124 @@ <h2>{{ subt }}</h2>
         <li class="commencement-date">
           {% blocktrans %}Not commenced{% endblocktrans %}
         </li>
-      {% elif not document.commencement_date %}
+      {% elif document.work.main_commencement and document.work.main_commencement.all_provisions %}
+        {% with document.work.main_commencement as commencement %}
+        {% with commencement.commencing_work as commencing_work %}
+        {% if not commencement.date %}
+          <li class="commencement-date">
+            {% blocktrans %}Commencement date unknown{% endblocktrans %}
+              {% if commencing_work %}
+                {% with numbered_title=commencing_work.numbered_title title=commencing_work.title %}
+                  {% work_resolver_url commencing_work as resolver_uri %}
+                  {% if numbered_title %}
+                    {% blocktrans %}– commenced by <a href="{{ resolver_uri }}">{{ numbered_title }}</a>{% endblocktrans %}
+                  {% else %}
+                    {% blocktrans %}– commenced by <a href="{{ resolver_uri }}">{{ title }}</a>{% endblocktrans %}
+                  {% endif %}
+                {% endwith %}
+              {% endif %}
+          </li>
+        {% else %}
+          <li class="commencement-date">
+            {% blocktrans with date=commencement.date|date:"j E Y" %}Commenced on {{ date }}{% endblocktrans %}
+              {% if commencing_work %}
+                {% with numbered_title=commencing_work.numbered_title title=commencing_work.title %}
+                  {% work_resolver_url commencing_work as resolver_uri %}
+                  {% if numbered_title %}
+                    {% blocktrans %}by <a href="{{ resolver_uri }}">{{ numbered_title }}</a>{% endblocktrans %}
```

Comment:
In the old template we did `title - numbered_title`

Reply:
What do you think of leaving them out? They can often be quite long or not very informative.

Reply:
I think that's fine, esp. since we provide a link.